.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "search.exclude": {
+        "coverage/": true
+    },
+}
```

Comment:
@connorshea Everyone should use vscode :P
https://github.com/danger/danger/pull/583/files#r80905047